### DISCUSSION TITLE
Q: How to get bucket size limit?

### DISCUSSION THREAD

**Discussion by quenbyako** [minio/minio#11652] [Q&A] [answered]
So, i've created small cluster of minio instances (for large amount of files) and i want to make storage balancer to all my instances. but i don't know how i can get available space for each bucket (or instance) in s3 api. Is it even possible?

**Comment by harshavardhana** [comment, marked as answer]
Bucket space is as much as your backend disks, there is no Limit on MinIO

**Reply by quenbyako** [reply]
But how i can get disks status through s3 api? or i need to create separated api?

**Reply by harshavardhana** [reply]
S3 API has no such provisions, you are talking about monitoring - use prometheus for that

**Reply by quenbyako** [reply]
Got it, thanks!
